extensions/wikia/PortableInfobox/PortableInfoboxHooks.class.php

```diff
@@ -3,21 +3,35 @@
 class PortableInfoboxHooks {
 	const PARSER_TAG_GALLERY = 'gallery';
 
-	static public function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
-		if ( F::app()->checkSkin( 'monobook', $skin ) ) {
-			Wikia::addAssetsToOutput( 'portable_infobox_monobook_scss' );
+	/**
+	 * adds portable infobox styles
+	 *
+	 * @param OutputPage $out
+	 * @param Skin $skin
+	 * @return bool
+	 */
+	static public function onBeforePageDisplay(OutputPage $out, Skin $skin) {
```

Comment:
Do we have a convention about writing `static public function ...` (1073 occurrences in app) or `public static function ...` (4180 occurrences in app)?